service/apis/qsl_service.go

```diff
@@ -75,6 +124,7 @@ func CreateFiltersQuery(filterlist string) (string, string, error) {
 		"<=": "le",
 		"<":  "lt",
 		"=":  "eq",
+		"!=": "eq",
```

Comment:
@kianjones4, late to the review party... but should the `!=` operator map to a different text code (ex: `ne`) than the `=` operator?

Reply:
nice catch. @formuzi could you fix that as well?

Reply:
Im not sure that there is an ne or neq function in dgraph, so i had != map to "not eq" later in that function